dds_web/api/user.py

```diff
@@ -118,7 +118,11 @@ def invite_user(args):
 
         # Append invite to unit if applicable
         if new_invite.role in ["Unit Admin", "Unit Personnel"]:
-            auth.current_user().unit.invites.append(new_invite)
+            if "Unit" in auth.current_user().role:
+                # Give new unit user access to all projects of the unit
+                auth.current_user().unit.invites.append(new_invite)
+                for project in auth.current_user().unit.projects:
```

Comment:
Doesn't looping though `auth.current_user().unit.projects` implicitly assume that every unit user holds the keys for every project ever created by a particular unit? 

Judged by the code, currently this procedure doesn't seem to be tolerant on missing keys (at least the `KeyNotFoundError` seems to be raised). What happens to the keys when a project is deleted or archived - should they persist nonetheless?

Reply:
Maybe the unit user can be given access to just active projects? So none that are 'Archived' or 'Deleted'?

Reply:
@MatthiasZepper @aanil The keys should be removed when the project is deleted or archived. So there won't be any keys for those projects.

Reply:
@aanil implemented the deletion of the keys last week https://github.com/ScilifelabDataCentre/dds_web/blob/ad24b040bab6450ab936b5a713d01fc38192e43c/dds_web/api/project.py#L339-L341

Reply:
@MatthiasZepper @aanil https://github.com/scilifelabdatacentre/dds_web/issues/874 
Not a must have though

Reply:
Added a check to see if the project is active.